pkg/controller/storageversionmigrator/storageversionmigrator.go

```diff
@@ -339,6 +357,24 @@ func (svmc *SVMController) sync(ctx context.Context, key string) error {
 	return nil
 }
 
+func (sv *SVMController) isResourceUpdatable(gvr schema.GroupVersionResource) (bool, error) {
```

Comment:
IMO this logic should be in the RV controller (probably before it sets the RV).

Reply:
Yeah good point, I was thinking that it made more sense to add here since this was the controller that actually kicked off the migration however since the first part of the process is adding the current RV it will do less work and we already pass the discovery client there. Let me see how that would look.